keyboards/planck/keymaps/vennjadin/keymap.c

```diff
@@ -0,0 +1,230 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "muse.h"
+#include "keymap_norwegian.h"
+#include "keymap_german_mac_iso.h"
+#include "keymap_icelandic.h"
+
+
+enum planck_layers {
+  _QWERTY,
+  _LOWER,
+  _RAISE,
+  _ADJUST
+};
+
+enum planck_keycodes {
+  QWERTY = SAFE_RANGE,
+  BACKLIT,
+};
+
+#define LOWER MO(_LOWER)
+#define RAISE MO(_RAISE)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* Qwerty
+ * ,-----------------------------------------------------------------------------------.
+ * | Esc  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Bksp |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Tab  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   Ø  |  Æ   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   Å  |Ent|shift|
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | 1pass| Ctrl | Alt  | GUI  |Lower |    Space    |Raise | GUI  | Alt  |Mute  |@     |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_QWERTY] = LAYOUT_planck_grid(
+    KC_ESC,      KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
+    KC_TAB,      KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    NO_OSTR, NO_AE,
+    KC_LSFT,     KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  NO_ARNG, KC_SFTENT,
+    LSG(KC_SPC), KC_LCTL, KC_LALT, KC_LGUI, LOWER,   KC_SPC,  KC_SPC,  RAISE,   KC_RGUI, KC_RALT, KC_MUTE,   NO_QUOT
+),
+
+/* Lower
+ * ,-----------------------------------------------------------------------------------.
+ * |      |   +  |   7  |   8  |   9  |   ,  |   %  | [    |  ]   |  =   |   _  |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |  -   |  4   |  5   |  6   |  .   |  /   |   (  |  )   | <    |   >  |  '   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |  0   |  1   |  2   |  3   |  *   |  \   |   {  |  }   |  #   |   $  |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
```

Comment:
The ASCII art implies <kbd>'</kbd> when in reality, it is `NO_PIPE`, and it also shows <kbd>$</kbd> when in reality it is `NO_CURR`, which is conventionally represented with the symbol ¤.